modules/sshkey.sh

```diff
@@ -29,15 +29,16 @@ function sshkey () {
       echo "Usage: $(basename "$0") sshkey delete \"<key>\""
       exit 1
     fi  
-    sed -i "/^$2/d" /root/.ssh/authorized_keys
+    sed -i "\|$2|d" /root/.ssh/authorized_keys
     if [ "$(detectrpi)" != "nonrpi" ]; then
-      sed -i "/^$2/d" /home/pi/.ssh/authorized_keys
+      sed -i "\|$2|d" /home/pi/.ssh/authorized_keys
     fi
   elif [ "$1" == "deleteall" ]; then
     rm /root/.ssh/authorized_keys
     if [ "$(detectrpi)" != "nonrpi" ]; then
       rm /home/pi/.ssh/authorized_keys
     fi
+    echo "all sshkeys are deleted."
```

Comment:
You could add a test statement before this line to check that the files don't exist to make sure they are actually deleted but this is optional (I learned this was good practice in school)